File: C99/1000-1099/1040-1049/1047.c
#include <stdio.h>
#include <math.h>

int main()
{
    int n1, n2, n3, n4, total, h, m;
    scanf("%d %d %d %d", &n1, &n2, &n3, &n4);
    total = (n3 * 60 + n4) - (n1 * 60 + n2);
    if (total < 1)
    {
        total += 24 * 60;
    }
    h = total / 60;
    m = total % 60;
    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", h, m);
    return 0;
}

File: python/1000-1099/1040-1049/1047.py
entry = input().split()
entry = [eval(i) for i in entry]

time = [0, 0]
total = (entry[2] * 60 + entry[3]) - (entry[0] * 60 + entry[1])
if total < 1:
    total += 24 * 60
time[0] = total // 60
time[1] = total % 60

print(f"O JOGO DUROU {time[0]} HORA(S) E {time[1]} MINUTO(S)\n");